core/shared/src/main/scala/com/lynbrookrobotics/potassium/streams/package.scala

```diff
@@ -0,0 +1,5 @@
+package com.lynbrookrobotics.potassium
+
+package object streams {
```

Comment:
I want the `Cancel` type alias to be available throughout the package, so I added it here.